showcase/app/components/mock/components/form/super-select/generic-content.gts

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
[question] would a simple approach like this one https://github.com/hashicorp/design-system/blob/main/showcase/app/components/mock/app/main/generic-text-content.gts work in this case?

Reply:
well that interface doesn't do anything it's not even attached to the template